.github/workflows/therock-ci-windows.yml

```
@@ -80,17 +75,17 @@ jobs:
       - name: Fetch sources
         run: |
           git config --global core.longpaths true
-          python ./TheRock/build_tools/fetch_sources.py --jobs 96 --no-include-math-libs
+          python ./TheRock/build_tools/fetch_sources.py --jobs 96 --no-include-rocm-systems
```

```suggestion
          python ./TheRock/build_tools/fetch_sources.py --jobs 96 --no-include-rocm-systems --no-include-rocm-libraries
```

There are several other subrepos you don't need to checkout out here (yet). Especially rocm-libraries is quite heavy and as you don't build it, there is no need to fetch it.